modules/process/manager_test.go

```diff
@@ -13,7 +13,10 @@ import (
 )
 
 func TestManager_Add(t *testing.T) {
-	pm := Manager{processes: make(map[int64]*Process)}
+	go func() {
+		_ = GetManager()
+	}()
+	pm := GetManager()
```

Comment:
The manager is not supposed to be the one you get from GetManager here.

Reply:
at least it wasn't ever intended to be ... 

I also don't quite understand the purpose of the goroutine...

Reply:
@zeripath just to trigger the race for unit tests. I can remove it since I could test that it prevents race localy with this already :)

Reply:
Since we have `sync.Once`, it should not trigger data race.